examples/web/index.html

```diff
@@ -127,8 +127,10 @@
         </div>
     </footer>
 
+    <script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
```

Comment:
https://electronjs.org/docs/faq#i-can-not-use-jqueryrequirejsmeteorangularjs-in-electron

Reply:
In theory you could call module.noConflict() to get back the original implementation, but cannot try it easily

Reply:
@psmulovics